event_model/tests/test_em.py

```diff
@@ -52,9 +52,9 @@ def test_compose_run():
     datum_doc = compose_datum(datum_kwargs={'slice': 5})
     event_doc = compose_event(
         data={'motor': 0, 'image': datum_doc['datum_id']},
-        timestamps={'motor': 0, 'image': 0}, filled={'image': False},
-        seq_num=1)
+        timestamps={'motor': 0, 'image': 0}, filled={'image': False})
     assert 'descriptor' in event_doc
+    assert event_doc['seq_num'] == 1
```

Comment:
You are making changes in the `compose_descriptor`, shouldn't it test the field exists in `bundle` above?

Reply:
The change is in the `event_counter` which is handed to `compose_event` we don't have access to the `event_counter` until we get to the `event` itself.

Reply:
I see now. Thanks for the explanation.